keyboards/eniigmakeyboards/ek87/keymaps/default_ansi/keymap.c

```diff
@@ -0,0 +1,11 @@
+#include QMK_KEYBOARD_H
```

Comment:
Keyboards **must** have a default keymap, which my suggestion is to rename the `default_ansi` folder to `default`

Reply:
Fixed